### PR TITLE
feat: Added LinkedIn Clone

### LINKED ISSUE #233: Feat: LinkedIn Clone
# Description:-
I want to add a LinkedIn clone with both frontend and backend

# Features:-
1. Authentication
2. Start a Post
3. Update a Post
4. Delete a Post
5. Add Connections
6. Like and Comment on a Post
7. Update Profile
8. See Other Profiles
9. Add a Profile Picture
10. Add Post Images
11. Search For Other Users

Please assign me this issue under IWOC

### PR DESCRIPTION
**Pull request by JiyaGupta-cs**
# Fixes Issue🛠️

Closes #233 

# Description👨‍💻 

I have added the LinkedIn Clone with a Readme

# Checklist✅ 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have added a demonstration in the form of a GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷

Uploading linkdein csdfjl.mp4…

### CODE REVIEW

File: Linkedin-clone/src/components/common/ProfileCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+              <p className="location">
+                {Object.values(currentProfile).length === 0
+                  ? `${currentUser.city}, ${currentUser.country} `
+                  : `${currentProfile?.city}, ${currentUser.country}`}
+              </p>
+            ) : (
+              <></>
+            )}
+            {currentUser.website || currentProfile?.website ? (
+              <a
```
[the remark this answers is not in the document]
Comment (JiyaGupta-cs) [reply Linkedin-clone/src/components/common/ProfileCard/index.jsx:92]: It is the part of website clone

**Comment by Kritika30032002** [comment]
update readme

**Comment by JiyaGupta-cs** [comment]
> update readme

@Kritika30032002 What else do I need to add to the readme?

**Comment by JiyaGupta-cs** [comment]
@Kritika30032002 
I have added the setup guidelines and the demo video to it
Please review the PR

**Comment by Kritika30032002** [comment]
Thanks for your Valuable contribution.
Don't forget to star the repo.

**Pull request merged**